viewer/src/migration/Cognite3DModel.ts

```diff
@@ -101,8 +123,15 @@ export class Cognite3DModel extends THREE.Object3D {
     })();
   }
 
-  resetNodeColor(_nodeId: number): void {
-    throw new NotSupportedInMigrationWrapperError();
+  resetNodeColor(nodeId: number): void {
```

Comment:
I think we should change signature to reflect that its async. It will still 'work' in old clients.

Reply:
Fair enough. The behavior is pretty much the same, except you will get access to the promise with your suggestion.